modules/docs/src/main/mdoc/docs/index.md

```diff
@@ -98,8 +98,6 @@ import cats.effect.Timer
 import scala.concurrent.ExecutionContext.global
```

Comment:
Please update the instructions higher up in this file on how to add the dependency in sbt. The `cats-effect` module is going away and the `cats-retry-core` module is changing name to just `cats-retry` (right?).

By the way that change of artifact name is going to mess with scala-steward. That's a shame, but not much we can do about it. We could publish a duplicate artifact as `cats-retry-core`, but we wouldn't want to do that forever. Eventually people will need to migrate.

Reply:
We're pushing some breaking changes, so it's not the worst thing :)